xml/admin_upgrade.xml

```diff
@@ -256,4 +256,213 @@ value: value)}'</command></screen>
+                  '73b92dd7816147058c3d0fbb67fb18f9',
+                  'admin']
+admin:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+73b92dd7816147058c3d0fbb67fb18f9:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+9b6d8d28393045c0914c959d0a5c0e33:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+12cda3c374144d74804298bdee4d686c:
+    SUSEConnect-0.3.11-3.15.1.x86_64
+jid:
+    20180809103558881056
+retcode:
+    0
+   </screen>
+   
+   <para>
+    If the package version is <literal>0.3.11-3.15.1</literal> (or higher) you
+    have the latest updates from the v2 channel installed.
+   </para>
+   </sect2>
+   
+   <sect2 xml:id="sec.deploy.upgrade.caasp2.timer">
+    <title>Disable Automatic <literal>transactional-update</literal></title>
+    <para>
+    To begin with the upgrade procedure, you first must disable the automatic
+    transactional update mechanism to avoid conflicts. To do so you must run a
+    <command>salt</command> across the nodes to disable the 
+    <literal>transactional-update.timer</literal>.
+    </para>
+    <screen>
+&prompt.user;<command>docker exec -i $(docker ps | grep salt-master | awk '{print $1}') \
+salt --batch 10 -P "roles:(admin|kube-(master|minion))" \
+cmd.run "systemctl disable --now transactional-update.timer"</command>
+
+Executing run on ['5f6688bbeac94d2ab5c4330dc7043fb2',
+                  'c3afd049edbe43afb4e2e5913a88291b',
+                  '5bf346291a18406290886c2e2f7c3e3f',
+                  'admin']
+
+5bf346291a18406290886c2e2f7c3e3f:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+admin:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+c3afd049edbe43afb4e2e5913a88291b:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+5f6688bbeac94d2ab5c4330dc7043fb2:
+    Removed symlink /etc/systemd/system/timers.target.wants/transactional-update.timer.
+jid:
+    20180807122220543037
+retcode:
+    0
+    </screen>
+   </sect2>
+   
+   <sect2 xml:id="sec.deploy.upgrade.caasp2.upgrade">
+    <title>Performing The Upgrade Procedure</title>
+    <para>
+     In the next step you must run the update command across your nodes.
+    </para>
+    <note>
+     <title>Batch size for upgrade</title>
+     <para>
+      In this example we have limited the number of nodes this step will be 
+      performed on to <literal>10 nodes</literal> at a time.
+     </para>
+      
+      <para>
+      This is a precaution to avoid problems on slower network connections.
+      If you are performing this step on a high bandwidth connection (for 
+      example from within the same datacenter as the cluster), you can raise the
```

Comment:
This sentence is close, but not quite right.

    .... If your cluster is hosted within a datacenter with a high bandwidth internet connection, you can increase this number. However, leaving it at the suggested batch size of 10 is recommended .... 

I added a "leave it" recommendation, because, the internet is unpredictable - so its better to limit the amount of nodes updating at once.

Reply:
Ok I will adjust that.